assets/src/components/reader-theme-selection/index.js

```diff
@@ -16,16 +16,36 @@ import { useContext, useMemo } from '@wordpress/element';
 import { ReaderThemes } from '../reader-themes-context-provider';
 import { Loading } from '../loading';
 import './style.css';
+import { AMPNotice } from '../amp-notice';
 import { ThemeCard } from './theme-card';
 
 /**
  * Component for selecting a reader theme.
  *
  * @param {Object} props Component props.
- * @param {boolean} props.disableCurrentlyActiveTheme Whether the currently active theme should be unselectable.
+ * @param {boolean} props.hideCurrentlyActiveTheme Whether the currently active theme should be unselectable.
  */
-export function ReaderThemeSelection( { disableCurrentlyActiveTheme = false } ) {
-	const { currentTheme, fetchingThemes, themes } = useContext( ReaderThemes );
+export function ReaderThemeSelection( { hideCurrentlyActiveTheme = false } ) {
```

Comment:
We should also hide the active theme in the onboarding wizard as well, or simply, set `hideCurrentlyActiveTheme` here to be `true` by default.

```diff
--- assets/src/onboarding-wizard/pages/choose-reader-theme/index.js	(revision fea953be34734834ef3b097bb31e9ce2574b1e0d)
+++ assets/src/onboarding-wizard/pages/choose-reader-theme/index.js	(date 1594337260089)
@@ -65,7 +65,7 @@
 
 	return (
 		<div className="choose-reader-theme">
-			<ReaderThemeSelection />
+			<ReaderThemeSelection hideCurrentlyActiveTheme={ true } />
 		</div>
 	);
 }
```

Reply:
Actually, we intentionally do show the active theme in the list in the wizard, and then switch the template moe from Reader to Transitional with a notice appearing on the summary screen to explain that. So I don't think we need to do this.

Reply:
Good point. I'm OK with this in that case.